prov/verbs/src/verbs_rma.c

```diff
@@ -276,22 +276,29 @@ fi_ibv_rdm_ep_rma_preinit(void **desc, void **raw_buf, size_t len,
 }
 
 static ssize_t
-fi_ibv_rdm_ep_rma_read(struct fid_ep *ep_fid, void *buf, size_t len,
-		    void *desc, fi_addr_t src_addr,
-		    uint64_t addr, uint64_t key, void *context)
+fi_ibv_rdm_ep_rma_readmsg(struct fid_ep *ep_fid, const struct fi_msg_rma *msg,
+		uint64_t flags)
 {
-	struct fi_ibv_rdm_ep *ep = container_of(ep_fid, struct fi_ibv_rdm_ep,
-						ep_fid);
+	if(msg->iov_count != 1 || msg->rma_iov_count != 1) {
+		assert(0);
+		return -FI_EMSGSIZE;
+	}
+
+	struct fi_ibv_rdm_ep *ep =
```

Comment:
While I agree mixed declarations/statements like this one should be converted to match the rest of the code base (and Linux kernel style guide), we definitely should not be worried about requiring common C99 compilation support.  For example, C99 designated initializers are an important feature that is widely supported and can greatly improve code safety.  I think that modern versions of MS Visual Studio even support them (>= VS2013? someone correct me if I'm wrong here).